docs/data-collection/notes-et.md

```diff
@@ -4,16 +4,45 @@
 
 !!! warning "To ensure accurate performance and prevent minor drifts in the pupil and corneal reflection (CR) thresholds, perform the ET Setup 10-15 minutes after powering the ET on."
 
-- [ ] Open the QCT experiment in *Psychopy* (`task-qct_bold.psyexp` file).
-    - [ ] Run the experiment by pressing the green play button.
-    - [ ] Press enter to begin the *camera setup mode*.
-    - [ ] The camera setup mode contains a view of one eye, and you can switch that view between two modes: one is the field-of-view of the ET, the second is an automatic zoom on the eye itself (or a random part if the eye is not visible).
-    - [ ] To ease the setup of the ET, switch to the field-of-view mode in the camera setup mode by pressing the <span class="keypress">:fontawesome-solid-caret-left:</span> or <span class="keypress">:fontawesome-solid-caret-right:</span> arrow.
-        This will allow you to adjust the ET position as the infrared camera provides feedback inside the scanning room on the projection screen.
-- [ ]  Enter the scanning room and adjust the ET arm position and focus of the lens until the eye is fully visible.
-    The **right** eye should be visible at the center of the coil's square window for the eye.
+- [ ] Initiate the fixation program on the stimuli laptop (*{{ secrets.hosts.psychopy | default("███") }}*), by typing the following on a terminal:
 
-    !!! important "Adjusting the camera's field-of-view and focus"
+    ??? warning "Make sure to have the correct environment loaded before invoking the task"
+
+        - [ ] Deactivate conda (if active):
+            ``` shell
+            conda deactivate
+            ```
+        - [ ] Load the new virtual environment:
+            ``` shell
+            source $HOME/psychopyenv/bin/activate
+            ```
+
+    ``` shell
+    cd ~/workspace/HCPh-fMRI-tasks
+    python {{ settings.psychopy.tasks.dwi }}.py
+    ```
+
+    A modal dialog will ask you for the number of trial (automatically calculated, DO NOT modify) and the session number.
+
+    !!! danger "The following two steps MUST be executed in this order"
+
+            - [ ] Drag and drop the modal dialog into the scanner's projector screen.
+            - [ ] Update the session number with the corresponding number.
+
+- [ ] Press <span class="keypress">Enter</span> to begin the *Camera Setup Mode*.
+
+    !!! tip "The *Camera Setup Mode* contains a view of one eye, and you can switch that view between two modes"
+
+        - The *Head Camera* will allow you to "find" and aim the right eye, as it has a wider field-of-view.
+        - The *{{ settings.eyetracker.eye | title }} Eye Camera* is a zoomed clip of the eye, if it is properly identified by the software (i.e., the eye is fully within the field-of-view of the camera).
```

Comment:
This view also makes it easier to check if the thresholding is good.

Reply:
Yup, that's why line 65 instructs to switch to this mode before checking the thresholding.